salt/metalk8s/addons/alert-logger/deployed/deployment.sls

```diff
@@ -0,0 +1,53 @@
+#!jinja | metalk8s_kubernetes
+
+{%- from "metalk8s/repo/macro.sls" import build_image_name with context %}
+
+{%- raw %}
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: metalk8s-alert-logger
+  namespace: metalk8s-monitoring
+  labels:
+    app: metalk8s-alert-logger
+    heritage: metalk8s
+    app.kubernetes.io/name: metalk8s-alert-logger
+    app.kubernetes.io/part-of: metalk8s
+    app.kubernetes.io/managed-by: salt
+spec:
+  selector:
+    matchLabels:
+      app: metalk8s-alert-logger
```

Comment:
This `app` label is "deprecated", we should rely on the `app.kubernetes.io/name` equivalent instead. Same in the Service selector.